timemachine/potentials/nonbonded.py

```diff
@@ -352,7 +369,7 @@ def nonbonded_on_specific_pairs(
     dij = distance_on_pairs(conf[inds_l], conf[inds_r], box, w_offsets)
     if cutoff is None:
         cutoff = np.inf
-    keep_mask = dij <= cutoff
+    keep_mask = dij < cutoff
```

Comment:
Was this an inconsistency between the C++ and the reference, or does this PR change the expectation about what we consider as an interaction?

Reply:
I think it was an inconsistency https://github.com/proteneer/timemachine/blob/99366d439cf608d22f56b5f071c189e980f8c52f/timemachine/cpp/src/kernels/k_nonbonded_pair_list.cuh#L122-L123